docs/recipes/create-immortal-spore.md

```diff
@@ -1,15 +1,24 @@
 # Create immortal spores on-chain
 
-Immortal is a Spore Extension that provides the following features:
-- An immortal spore will live on-chain forever and cannot be destroyed 
-- Spore Extensions can only be added on a spore on the spore's creation
+## What is immortal
 
-Immortal is a initial extension that comes with the Spore Protocol, which is why we call it Spore Core Extension. 
+`Immortal` is a Spore Extension that offers the following rules to spores:
 
-If you want to create a spore with immortal extension enabled, you need to pass the `immortal` parameter to the props of the `createSpore` API when calling it:
+1. Immortal extension is enabled for a spore if the spore has a `immortal=true` in its `SporeData.contentType`
+2. An immortal spore lives on-chain forever, and cannot be destroyed under any circumstances
+
+To create a spore with the immortal extension enabled, you need to pass the `immortal` parameter to the props of the `createSpore` API when calling it. After successfully creating an immortal spore, any attempt by the owner to destroy it will result in a failed transaction due to verification by the `SporeType` script.
+
+## Create immortal spores
+
+There are two recommended ways to set the `immortal` parameters while creating a spore. Note that both ways are fine and can complete the mission nicely, so choose whichever you prefer.
+
+There are two recommended ways to set the `immortal` parameters when creating a spore. Note that both ways are equally valid and will accomplish the mission successfully, so choose whichever method you prefer.
+
+### Set in the `contentTypeParameters` object
```

Comment:
// Can “set in” be understood as "define"/"specify"? Better replace it with one verb more accurate